src/components/ViewNote/ViewNote.css

```diff
@@ -17,4 +17,61 @@
 
 .card_icons:hover {
     cursor: pointer;
+}
+
+.dropdown {
+    position: relative;
+    display: inline-block;
+}
+
+.dropdown-content{
+  position: absolute;
+  display: flex;
+  flex-flow: row wrap;
+  gap: 10px;
+  background-color: #f9f9f9;
+  min-width: 13rem;
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+  padding: 12px 16px;
+  z-index: 1;
+}
+
+.color_icon {
+    height: 24px;
+    width: 24px;
```

Comment:
here also

Reply:
i am using px because i want these exact value on each width - Dont want to change the size of it.